### ISSUE TITLE
Imrpove Bug template of version

### ISSUE THREAD

**Issue by ipajen** [aspirepress/AspireUpdate#190]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

Version
What version of AspireUpdate are you running?
0.5 (default)
1.0
Nightly

What is nightly, it could be huge difference if its main, translate, playground-ready bransch. 

Maybe add the fixed "main" , "playground-ready" and a new "other" for other like temp translation?

### Mockups or Images of the feature

NA

**Comment by asirota** [comment]
We need to eventually rename the branches. I want development nightly and stable.

**Comment by afragen** [comment]
I'm assuming we would also have feature branches as well.

**Comment by asirota** [comment]
Need some eyes on this @namithj @costdev --- what do you think about changing the branch names to be more respective of our core workflow:

- `main` -> `development`
- new branch called `nightly` which would be a pull  from `development` into `nightly` every night at say 11pm Eastern Time (or some time we decide together)
-  `playground-ready` -> `stable`

This way PRs would go into `development` on a daily basis.

`nightly` would be a build someone could use to test the status of the current workflow.

`stable` would be the branch we cut into a release every time we complete a sprint/feature push.

Thoughts?

**Comment by ipajen** [comment]
Which of the branches could be open for merge, so @asirota don't need to accept all merges on each bransch.

**Comment by asirota** [comment]
We only do PR on main here. I also sometimes fix issue templates off the issue template branc

**Comment by asirota** [comment]
And of course translations.

**Comment by afragen** [comment]
`translations` is just a feature branch and should be removed once the feature is merged.

**Comment by asirota** [comment]
Should we change the playgrounds blueprint to include test-reports plugin?

**Comment by asirota** [comment]
Should we change the playgrounds blueprint to include test-reports plugin?

**Comment by ipajen** [comment]
> Should we change the playgrounds blueprint to include test-reports plugin?

Yes would be great, also if there is a plugin to update `wp-config.php` would be great to include.

**Comment by afragen** [comment]
You can actually use the [WP Debugging](https://wordpress.org/plugins/wp-debugging) plugin to add things to `wp-config.php` via a filter.
